listenbrainz/webserver/static/js/src/stats/UserStats.tsx

```diff
@@ -0,0 +1,130 @@
+import * as ReactDOM from "react-dom";
```

Comment:
remind me, were we naming this "Reports"? If so, we should fix the name of the file and the component

Reply:
Yeah it's on my mind, was gonna do it at the end.